public/src/components/channelManagement/variantEditorSeparateArticleCountEditor.tsx

```diff
@@ -2,6 +2,12 @@ import React from 'react';
 import FormControlLabel from '@material-ui/core/FormControlLabel';
 import Checkbox from '@material-ui/core/Checkbox';
 import { SeparateArticleCount } from '../../models/epic';
+import { TextField } from '@material-ui/core';
+import { useForm } from 'react-hook-form';
+
+interface FormData {
+  copy: string;
```

Comment:
If you clear the contents of this field does the tool submit an empty string (rather than null) to the database? I'm wondering what then happens in the epic

Reply:
That makes sense. Perhaps it's worth being defensive and transforming `''` to `undefined` in this component?